src/fill.vertex.glsl

```diff
@@ -10,15 +10,6 @@ attribute vec2 a_pos;
 
 uniform mat4 u_matrix;
 
-#ifdef MAPBOX_GL_JS
-#pragma mapbox: define color lowp
```

Comment:
Can you please investigate and respond to @lucaswoj's comment [here](https://github.com/mapbox/mapbox-gl-shaders/pull/4#discussion_r65236853) about why these may actually be necessary?